lib/instrumentation/modules/@apollo/server.js

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and other contributors where applicable.
+ * Licensed under the BSD 2-Clause License; you may not use this file except in
+ * compliance with the BSD 2-Clause License.
+ */
+
+'use strict'
+
+const shimmer = require('../../shimmer')
+
+module.exports = function (apolloServerCore, agent, { enabled }) {
```

Comment:
style nit: Can we s/apolloServerCore/apolloServer/g to match the new module name, please?

Reply:
done in commit 47ce659b